store/sqlstore/channel_member_history_store.go

```diff
@@ -161,26 +162,41 @@ func (s SqlChannelMemberHistoryStore) getFromChannelMembersTable(startTime int64
 }
 
 func (s SqlChannelMemberHistoryStore) PermanentDeleteBatch(endTime int64, limit int64) (int64, error) {
-	var query string
+	var (
+		query string
+		args  []interface{}
+		err   error
+	)
+
 	if s.DriverName() == model.DATABASE_DRIVER_POSTGRES {
-		query =
-			`DELETE FROM ChannelMemberHistory
-				 WHERE ctid IN (
-					SELECT ctid FROM ChannelMemberHistory
-					WHERE LeaveTime IS NOT NULL
-					AND LeaveTime <= :EndTime
-					LIMIT :Limit
-				);`
+		var innerSelect string
+		innerSelect, args, err = s.getQueryBuilder().
+			Select("ctid").
+			From("ChannelMemberHistory").
+			Where(sq.And{
```

Comment:
I like how you are doing it here. Can the others be done similar to how you are doing it here (using 1 .where and a .and)